spec/clustering/cluster_ht_item_spec.rb

```diff
@@ -2,18 +2,17 @@
 
 require "spec_helper"
 require "clustering/cluster_ht_item"
-RSpec.xdescribe Clustering::ClusterHtItem do
+RSpec.describe Clustering::ClusterHtItem do
   let(:item) { build(:ht_item) }
```

Comment:
i'd hard-code the ocns for the items where the tests are relying on them being different

Reply:
Cleaned up and clarified OCN usage here.